_data/accessibility-tests/character-count.yml

```diff
@@ -0,0 +1,68 @@
+title: Character count
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain the field's input requirements.
+    summary_additional: |
+      The text field has a visible label above it to indicate its purpose. For example, "Enter your full name."
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.3.5
+  - summary: Users can see and hear context about character count errors.
+    summary_additional: |
+      Clear, readable information about character count appears below the text field.
+      When you use a screen reader to tab to the text field, you hear accurate information about how many characters remain for the user to enter.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Content remains visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      text stays in position inside the text field, and the character count keeps its position below the text field.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.4
+  - summary: Formatting remains when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      the character count keeps its formatting, and sufficient space remains between the character count and the text field above it.
+    test_status: pass
+    test_type: zoom
```

Comment:
> [!note]
> This was marked as a "design" test type in the spreadsheet, but I moved it to zoom since the test curently requires zoom to test. We can move it to a future "text size/formatting" category when we have one.